app/views/users/show.html.erb

```diff
@@ -1,29 +1,76 @@
-<div>
-<div class="u-container">
-<%= image_tag @user.photo || "", class: "empty-square" %>
-    <div class="u-info">
-    <h3><%= link_to @user.name, @user %></h3>
+<style>
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
 
-      <br>
-      Number of posts: <%= @user.posts_counter %>
-    </div>
-</div>
- <div class="bio">
-      <h5>Bio</h5>
-      <p><%= @user.bio %></p>
-  </div>
-  <h2>Recent Posts</h2>
-<% Post.limit(3).each do |post| %>
-  <div class="post-preview">
-    <h3><%= post.title %></h3>
-    <p><%= post.text %></p>
-    <p>Number of comments: <%= post.comments.count %></p>
-    <p>Number of likes: <%= post.likes_counter %></p>
-  </div>
-<% end %>
+.card-body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  padding: 0;
+}
+
+.card-text {
+  margin: 0;
+}
 
+.recent-post {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  border: 1px solid #ccc;
+  padding: 10px;
+  margin-bottom: 10px;
+}
 
-  <% if Post.count > 3 %>
-    <%= link_to "Show All Posts", user_posts_path(@user) %>
-  <% end %>
+.recent-post p {
+  margin-bottom: 5px;
+}
+
+.card-img-top {
+    border-radius: 50%;
+    width: 200px;
+    height: 200px;
+    object-fit: cover;
+  }
+</style>
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** Good job adding all the changes recommended by the previous reviewer. However I think it would be better if you move all your styling code in a specific styling file inside your `app/assets/stylesheets/` directory. This for best practices reasons.